scss/_maps.scss

```diff
@@ -57,11 +57,30 @@ $background-colors: map-merge(
 ) !default;
 // End mod
 
+$theme-colors-dark: null !default;
+$theme-colors-rgb-dark: null !default;
 $theme-colors-text-dark: null !default;
 $theme-colors-bg-subtle-dark: null !default;
 $theme-colors-border-subtle-dark: null !default;
 
 @if $enable-dark-mode {
+  // scss-docs-start theme-color-dark-map
+  $theme-colors-dark: (
```

Comment:
`$theme-colors` is defined in `_variables.scss`. If we want to respect a certain logic, is it possible to define it in `_variables-dark.scss` instead?

Note: `$theme-colors-rgb-dark` can stay there.